policyengine_us/parameters/gov/states/dc/tax/income/credits/ptc/fraction_elderly.yaml

```diff
@@ -5,17 +5,25 @@ metadata:
   threshold_unit: currency-USD  # federal AGI
   amount_unit: /1
   reference:
+    - title: Code of the District of Columbia § 47–1806.06
+      href: https://code.dccouncil.gov/us/dc/council/code/sections/47-1806.06
     - title: 2021 Form D-40 Booklet
       href: https://otr.cfo.dc.gov/sites/default/files/dc/sites/otr/publication/attachments/52926_D-40_12.21.21_Final_Rev011122.pdf#page=49
     - title: 2022 Form D-40 Booklet
       href: https://otr.cfo.dc.gov/sites/default/files/dc/sites/otr/publication/attachments/2022_D-40_Booklet_Final_blk_01_23_23_Ordc.pdf#page=47
+    - title: 2023 Form D-40 Booklet
+      href: https://otr.cfo.dc.gov/sites/default/files/dc/sites/otr/publication/attachments/2023_D40_Book_Final_012324.pdf#page=47
 brackets:
   - threshold:
       2021-01-01: -.inf
     amount:
       2021-01-01: 0.03
   - threshold:
-      2021-01-01: 76_700
-      2022-01-01: 78_600
+      values:
+        2021-01-01: 76_700
+        2022-01-01: 78_600
+        2023-01-01: 83_700
+      metadata:
+        uprating: gov.irs.uprating
```

Comment:
check if we need rounding here, based on legal code

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/9fdca0f3-13c9-4f37-8275-9fe03e101d4d)